engine/core/src/main/java/org/datacleaner/util/batch/BatchRowCollectingTransformer.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.util.batch;
+
+import java.util.Collection;
+
+import javax.inject.Inject;
+
+import org.datacleaner.api.Close;
+import org.datacleaner.api.Initialize;
+import org.datacleaner.api.InputRow;
+import org.datacleaner.api.OutputRowCollector;
+import org.datacleaner.api.Provided;
+import org.datacleaner.api.Transformer;
+
+/**
+ * Added by jakub on 20.10.15
```

Comment:
A bit of javadoc here would be nice. Took me a while to figure out the difference between this class and ```BatchTransformer```. In the end I guess it is that this one supports one-to-n transformations using an ```OutputRowCollector```.